frontend/src/types/ModelTypes.ts

```diff
@@ -3,7 +3,9 @@ export interface Keyword {
     name: string
     doc: string
     args: string | null
+    tags: string | null
```

Comment:
should be `string[]` instead, tags_string could be turned into method using tags field

Reply:
ok, should it be everywhere instead of string | null?